packages/next/bin/next.ts

```diff
@@ -14,12 +13,12 @@ import arg from 'arg'
 })
 
 const defaultCommand = 'dev'
-const commands = [
-  'build',
-  'start',
-  'export',
-  defaultCommand,
-]
+const commands: {[command: string]: () => Promise<{}>} = {
+  build: () => import('./next-build'),
```

Comment:
I think this should be `require()` as it's Node.js and not webpack'ed, so I'm not entirely sure the Typescript output will be correct in this case.

Reply:
Just checked the output, seems like it's definitely better to use `require()` in this case.

```
const commands = {
    build: () => Promise.resolve().then(() => __importStar(require('./next-build'))),
    start: () => Promise.resolve().then(() => __importStar(require('./next-start'))),
    export: () => Promise.resolve().then(() => __importStar(require('./next-export'))),
    dev: () => Promise.resolve().then(() => __importStar(require('./next-dev'))),
};
```